core/src/engine/coro/pool.hpp

```diff
@@ -184,6 +236,27 @@ typename Pool<Task>::Coroutine Pool<Task>::CreateCoroutine(bool quiet) {
   }
 }
 
+template <typename Task>
+bool Pool<Task>::TryPopulateLocalCache() {
+  // using initial_coroutines_.size_approx() fast
+  // because it has one producer queue
+  const std::size_t deque_num =
```

Comment:
But I can add a counter for the number of coroutines in initial_coroutines_

Reply:
No need do so, the comment is right